python/tvm/__init__.py

```diff
@@ -68,6 +68,13 @@
 # Contrib initializers
 from .contrib import rocm as _rocm, nvcc as _nvcc, sdaccel as _sdaccel
 
+try:
+    if support.libinfo()["USE_MICRO"] != "NOT-FOUND":
+        from . import micro
+except support.LibInfoUnavailableError:
+    pass
+
```

Comment:
Libinfo will now always return a dictionary, so update this to check that the dictionary contains USE_MICRO.